plume-common/src/activity_pub/mod.rs

```diff
@@ -129,38 +130,38 @@ where
         .sign(sender)
         .expect("activity_pub::broadcast: signature error");
 
+    let mut rt = tokio::runtime::current_thread::Runtime::new().expect("new rt");
+    let client = ClientBuilder::new()
+        .connect_timeout(std::time::Duration::from_secs(5))
+        .build()
+        .expect("Can't build client");
     for inbox in boxes {
         // TODO: run it in Sidekiq or something like that
```

Comment:
is this comment still relevant?

Reply:
Not really, but it is for the background jobs in general, for the moment we just spawn them in a separate thread and that's it. I will remove it and create a more general issue.